blog/updating_our_tracking_prevalence_metrics.md

```diff
@@ -0,0 +1,98 @@
+title: Updating our tracking prevalence metrics
+subtitle: Making metrics that make more sense.
+author: privacy team
+type: article
+publish: True
+date: 2019-02-22
+tags: blog, update
+header_img: blog/updating-prevalence-metrics/header.jpg
++++
+
+This month we updated the site with data from over 800 million page loads during January. In this
+data release we have also made a change to one of the main figures that we publish - `site reach`.
+Metrics are as good as their ability to capture a simplified version of reality. The
+main motivation for redefining `site reach` attempts to achieve exactly that - simplify the
+understanding of prevalence.
+
+The site reach stat was conceived as a measure of how many different sites a tracker has some
+presence. In contrast to `reach` - the proportion of pages loading a tracker - it shows how spread
```

Comment:
"how many different sites a tracker has some presence", what about "**on** how many different sites a tracker is present"?